cvat/apps/dataset_manager/tests/test_rest_api_formats.py

```diff
@@ -5,29 +5,36 @@
 import copy
 import json
 import os.path as osp
+import os
+import av
+import numpy as np
 import random
 import xml.etree.ElementTree as ET
 import zipfile
 from io import BytesIO
+import itertools
 
 from datumaro.components.dataset import Dataset
-from datumaro.util.test_utils import TestDir, compare_datasets
+from datumaro.util.test_utils import compare_datasets, TestDir
 from django.contrib.auth.models import Group, User
 from PIL import Image
 from rest_framework import status
 from rest_framework.test import APIClient, APITestCase
 
+import cvat.apps.dataset_manager as dm
 from cvat.apps.dataset_manager.bindings import CvatTaskDataExtractor, TaskData
 from cvat.apps.dataset_manager.task import TaskAnnotation
+from cvat.apps.dataset_manager.annotation import TrackManager
```

Comment:
@MashaSS , the import is unused for now. It leads to a warning from pylint. Could you please fix?

Reply:
Done.